test/unit/org/apache/cassandra/db/CellTest.java

```diff
@@ -103,6 +105,26 @@ public void testConflictingTypeEquality()
         }
     }
 
+    @Test
+    public void testUnmarshallableInMulticellCollection()
+    {
+        List<CQL3Type.Native> unmarshallableTypes = new ArrayList<>();
+        for (CQL3Type.Native nativeType : CQL3Type.Native.values())
+        {
+            ColumnMetadata c = fakeColumn("c", MapType.getInstance(Int32Type.instance, nativeType.getType(), true));
+            BufferCell cell = BufferCell.tombstone(c, 0, 4, CellPath.create(ByteBufferUtil.bytes(4)));
+            try
+            {
+                Assert.assertTrue(cell.toString().contains("tombstone"));
```

Comment:
nit: would be better if asserting on the format, instead of just contains.

Reply:
thanks for the review, @yifan-c. I pushed a new commit addressing your feedback